packages/core/documentation/colors.mdx

```diff
@@ -18,3 +18,11 @@ eller om du har behov for flere variabler enn farger:
 Klikk på fargene for å kopiere scss-variabelnavnet til clipboard
 
 <Colors />
+
+## Farger som util klasser
+
+`@import "~@fremtind/jkl-core/colors-util.scss";`
+
+farger som util klasser
+
+Disse klassene er strengt tatt ut og brukes i dokumentasjon der det er behov, bruker vanligvis ikke disse klassene, men først og fremst fargevariablene ovenfor.
```

Comment:
Er ikke enig i denne setningen.

Jeg har i mange tilfeller savner å ha fargene som utility-klasse, som spacing. Hvis jeg kan komme meg unna å mått lage en css/scss/less fil i komponenten min, fordi det eneste stil jeg trenger er spacing og farge, så vil jeg gjerne bruke de.

Feks:
```
<div className="jkl-spacing--bottom-2 jkl-color-info--darkbg jkl-bgcolor-svart">Litt info her</div>
```

Jeg kommer til å bruke dette. Jeg syns vi skal fortsette i denne retningen også, veldig mange ting kan vi komme oss unna på klientside ved å legge på utility-klasse isteden for å skrive css. Ta en titt på https://tailwindcss.com

Reply:
Jeg er usikker på om vi skal prøve å være Tailwind, siden vi ikke har bygget opp designsystemet med tanke på å være et utility-first CSS-rammeverk. Jeg tror imidlertid at det kan være fint å ha inn elementer av det, og jeg tor at fargene er en av tingene vi fint kan eksponere på denne måten.

Litt friksjon under utvikling er ikke nødvendigvis alltid en dårlig ting. Stort sett vil jeg si det er _bra_ at man blir tvunget til å tenke to ganger over hva man forsøker å lage, og hvorfor. Der tror jeg løsningen med å bruke stilark og variabler trumfer utility-klasser. Siden designsystemet er mer enn bare samlingen av egenskaper og variabler bør disse tingene så langt som mulig sees i sammenheng av både hverandre og helheten, og stilark med BEM—slik vi har—tvinger deg til å gjøre dette i større grad.

Om et element kun trenger å få satt margin og bakgrunnsfarge på seg—som i eksemplet ditt—ville jeg nesten ikke kalt det en komponent en gang, og i slike tilfeller synes jeg uility-klassene kommer til sin rett. Vi må bare passe på at det ikke blir _for_ enkelt å slenge sammen noe som ikke er gjennomtenkt eller passer inn i designsystemet.